modules/api_reference/src/Plugin/UnsupportedExtensionException.php

```diff
@@ -0,0 +1,8 @@
+<?php
+
+namespace Drupal\devportal_api_reference\Plugin;
+
+/**
+ * Defines an unsupported extension exception for Devportal API Reference.
+ */
+class UnsupportedExtensionException extends \Exception { }
```

Comment:
Module-specific exceptions should be used like this:
https://github.com/Pronovix/password_enhancements/tree/8.x-1.x/src/Exception
https://github.com/Pronovix/password_enhancements/issues/44 (including this fix)